.stickler.yml

```diff
@@ -0,0 +1,3 @@
+linters:
+  stylelint:
+    config: 'stylelint.config.js'
```

Comment:
### :thinking: Have you investigate the reason why the linter is not working? The config file looks good to me.

#### I ran the [`stylelint`](https://stylelint.io/) [locally](https://github.com/microverseinc/linters-config/tree/master/css#troubleshooting) and caught no error/warning. :+1:

Reply:
Hi Bola,

Thank you for your feedback. I'm not sure whats the issue, I'll see if it happens with other repositories in the future. I also ran stylelint locally and no issues were found.

Greetings
@mariobarrioss